src/lib/table/table.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ChangeDetectionStrategy, Component, ViewEncapsulation} from '@angular/core';
+import {CdkTable} from '../core/data-table/data-table';
+
+/**
+ * Wrapper for the CdkTable with Material design styles.
+ */
+@Component({
+  moduleId: module.id,
+  selector: 'md-table, mat-table',
+  templateUrl: '../core/data-table/table.html',
+  styleUrls: ['table.css'],
+  host: {
+    'class': 'mat-table',
+  },
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MdTable<T> extends CdkTable<T> { }
```

Comment:
Ahh right, should probably add the test to git before I commit =P There's a test now